spec/features/mi_bridges_driving_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.feature "MI Bridges Driving" do
   # to run this in a non-headless way, remove the `:js` flag below
-  scenario "successfully drives, then re-drives a SNAP App", :driving do
+  scenario "successfully drives, then re-drives a SNAP App", :js, :driving do
```

Comment:
Why do we need `:js` here?

Reply:
so it runs on a headless browser by default

Reply:
👍